tests/tests.py

```diff
@@ -0,0 +1,92 @@
+try:
```

Comment:
I'd suggest renaming this module to test_service.py. Don't think you are planning to put all the tests into one file... I hope so:)